website/content/en/docs/faqs/_index.md

```diff
@@ -141,4 +141,4 @@ Now run `make manifests` to update your `role.yaml`.
 [rbac-markers]: https://book.kubebuilder.io/reference/markers/rbac.html
 [rbac]:https://kubernetes.io/docs/reference/access-authn-authz/rbac/
 [scorecard-doc]: https://sdk.operatorframework.io/docs/advanced-topics/scorecard/
-[basic-project-doc]: https://book.kubebuilder.io/cronjob-tutorial/basic-project.html
\ No newline at end of file
+[project-doc]: /docs/overview/project-layout
```

Comment:
project doc should have an actual link right?

Reply:
It is how we link the docs in the docs with relative paths. 
We cannot add the full link because it will point for master and we want the docs release according to the tags.

Reply:
Able to preview the changes using netlify.